modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/internal/search/CommercePriceListIndexer.java

```diff
@@ -302,6 +304,8 @@ protected Document doGetDocument(CommercePriceList commercePriceList)
 				"Document " + commercePriceList + " indexed successfully");
 		}
 
+		_commercePriceListLocalService.cleanPriceListCache();
```

Comment:
@dantewang Why you put it here rather than the reindex method?

Reply:
@tinatian This method is called by other reindex methods, but I agree that it looks confusing. I'll update the pull to put the clean up in reindex methods tomorrow.